src/components/Common/UI/DatePicker/DatePicker.tsx

```diff
@@ -83,7 +76,9 @@ export const DatePicker = ({
   const { container } = useTheme()
 
   // Convert JavaScript Date to DateValue for internal use
-  const internalValue = dateToCalendarDateValue(value)
+  // Format the date as YYYY-MM-DD for parseDate
+  const formattedDate = value ? value.toISOString().split('T')[0] : ''
```

Comment:
Should this be refactored to a function named `formatForParseDate` to remove the need for this comment?

Reply:
Implemented